wiki/en/Software-Manual.md

```diff
@@ -146,6 +146,10 @@ This changes the visual style of the audio meters, independently of the chosen s
 
 This sets the number of rows displayed in the server audio mixer, for use with larger ensembles.
 
+### Audio Alerts
+
+This turns on a sound alert for when someone joins a chat, or when receiving a new chat message.
```

Comment:
It also gives a message if someone joins and doesn't send a chat

Reply:
I think that's implied, no?

Reply:
I think he means that there's an alert when someone joins a server, not a chat. Then there's another alert for new chat messages.